docs/rm/snax_cluster.md

```diff
@@ -22,50 +22,50 @@ There are three modifications in the SNAX cluster:
 
 The figure below shows a zoomed-in visualization highlighting the changes in the original Snitch micro-architecture:
 
-TODO - ADD my figure
+![SNAX modifications](https://drive.google.com/uc?id=11dr-GN-QwNBFIywlYFiX9Mdg9COQ_2vU)
```

Comment:
It looks like there's a stray title in this figure?

Reply:
@JosseVanDelm  EZ fix, by just re-uploading in google drive! No need for commits nyaahaa